x/evidence/keeper/infraction_test.go

```diff
@@ -1,177 +0,0 @@
-package keeper_test
```

Comment:
I don't think mocks test makes sense here for this function. As everything would be mocked. Unless someone sees a value.